docs/VnVPlan/VnVPlan.tex

```diff
@@ -197,6 +223,8 @@ \subsection{Software Validation Plan}
 
 \wss{This section might reference back to the SRS verification section.}
 
+Validation of the project is critical and our supervisor will be vital for this process. Our supervisor will make sure that our Software solves the problem and it solves the user groups needs. Our supervisor is in an excellent position to validate the software as not only are they a domain expert they also fit the potential user group.
```

Comment:
rephrase second line to be 'Our supervisor will make sure that our software solves the problem and the user group needs.' Add a comma after 'expert' in next line.